bookstore/bookstore.py

```diff
@@ -1,29 +1,45 @@
 def create_bookstore(name):
-    pass
-
+    return {'name': name, 'authors': [], 'books':[]}
+    
 def add_author(bookstore, name, nationality):
-    pass
+    author = {'name': name, 'nationality': nationality, 'id':len(bookstore['authors'])}
+    bookstore['authors'].append(author)
+    return author
 
 
 def get_author_by_name(bookstore, name):
-    pass
+    for author in bookstore['authors']:
+        if author['name'] == name:
+            return author
 
 
 def get_author_by_id(bookstore, author_id):
-    pass
+    for author in bookstore['authors']:
+        if author['id'] == author_id:
+            return author
 
 
 def add_book(bookstore, title, isbn, author_id):
-    pass
+    book = {'title': title, 'isbn': isbn, 'author_id': author_id, 'id':len(bookstore['books'])}
```

Comment:
Your dictionary definitions are getting a bit long for a single line, your code would be more readable if you were to multi-line it like this:
```
book = {'title': title,
        'isbn': isbn,
        'author_id': author_id,
        'id': len(bookstore['books'])}